docs/concepts/executor/executor-in-flow.md

```diff
@@ -0,0 +1,146 @@
+    DocumentArray(...),  # from Executor2
+    DocumentArray(...),  # from Executor3
+]
+```
+
+- Use `docs_map` to receive a Dict, where each item's key is the name of an upstream Executor and the value is the DocumentArray coming from that Executor:
+
+```python
+{
+    'Executor1': DocumentArray(...),
+    'Executor2': DocumentArray(...),
+    'Executor3': DocumentArray(...),
+}
+```
+
+(no-reduce)=
+### Reducing multiple DocumentArrays to one DocumentArray
+
+The `no_reduce` argument determines whether DocumentArrays are reduced into one when being received:
+
+- To reduce all incoming DocumentArrays into **one single DocumentArray**, do not set `no_reduce` or set it to `False`. The `docs_map` and `docs_matrix` will be `None`.
+- To receive **a list all incoming DocumentArrays** set `no_reduce` to `True`. The Executor will receive the DocumentArrays independently under `docs_matrix` and `docs_map`.
+
+```python
+from jina import Flow, Executor, requests, Document, DocumentArray
+
+
+class Exec1(Executor):
+    @requests
+    def foo(self, docs, **kwargs):
+        for doc in docs:
+            doc.text = 'Exec1'
+
+
+class Exec2(Executor):
+    @requests
+    def foo(self, docs, **kwargs):
+        for doc in docs:
+            doc.text = 'Exec2'
+
+
+class MergeExec(Executor):
+    @requests
+    def foo(self, docs_matrix, **kwargs):
+        documents_to_return = DocumentArray()
+        for doc1, doc2 in zip(*docs_matrix):
+            print(
+                f'MergeExec processing pairs of Documents "{doc1.text}" and "{doc2.text}"'
+            )
+            documents_to_return.append(
+                Document(text=f'Document merging from "{doc1.text}" and "{doc2.text}"')
+            )
+        return documents_to_return
+
+
+f = (
+    Flow()
+    .add(uses=Exec1, name='exec1')
+    .add(uses=Exec2, name='exec2')
+    .add(uses=MergeExec, needs=['exec1', 'exec2'], no_reduce=True)
+)
+
+with f:
+    returned_docs = f.post(on='/', inputs=Document())
+
+print(f'Resulting documents {returned_docs[0].text}')
+```
+
+
+```shell
+────────────────────────── 🎉 Flow is ready to serve! ──────────────────────────
+╭────────────── 🔗 Endpoint ───────────────╮
+│  ⛓     Protocol                    GRPC │
+│  🏠       Local           0.0.0.0:55761  │
+│  🔒     Private     192.168.1.187:55761  │
+│  🌍      Public    212.231.186.65:55761  │
+╰──────────────────────────────────────────╯
+
+MergeExec processing pairs of Documents "Exec1" and "Exec2"
+Resulting documents Document merging from "Exec1" and "Exec2"
+```
+
+## Call a Flow from an Executor
+
+To call other another Jina Flow using `Client` from an `Executor`, you also need to use `async def` and async Client.
+
+```python
+from jina import Client, Executor, requests, DocumentArray
+
+
+class DummyExecutor(Executor):
+    c = Client(host='grpc://0.0.0.0:51234', asyncio=True)
+
+    @requests
+    async def process(self, docs: DocumentArray, **kwargs):
+        self.c.post('/', docs)
```

Comment:
If u use async then use `await`

Reply:
I am not sure this section makes so much sense

Reply:
I agree, removed